pom.xml

```diff
@@ -2305,6 +2305,12 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-wrapper-plugin</artifactId>
+                    <version>3.2.0</version>
```

Comment:
is this forcing a newer version? or what is this for?

Reply:
This is for maven 4.0 which requires that when wrapper is used, explicit version is defined.